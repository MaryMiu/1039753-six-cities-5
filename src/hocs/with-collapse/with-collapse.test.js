import React from 'react';
import renderer from 'react-test-renderer';
import withCollapse from './with-collapse';

const MockComponent = () => {
  return <div />;
};

const MockComponentWrapped = withCollapse(MockComponent);

it(`withCollapse is render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      opened={true}
      onClickToggle={() => {}}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
