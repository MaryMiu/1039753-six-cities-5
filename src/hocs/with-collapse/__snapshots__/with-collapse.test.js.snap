// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withCollapse is render correctly 1`] = `<div />`;
